src/components/Guide/CategoryDescriptions.js

```diff
@@ -0,0 +1,60 @@
+import React from 'react'
+import styled from 'styled-components/macro'
+
+export default function CategoryDescriptions() {
+  return (
+    <Categories>
+      <Description>
+        <Category>Personal time</Category>
+        <span>
+          Pack your hobbies and that bullet point from your bucket list!
+        </span>
+      </Description>
+      <Description>
+        <Category>Self time</Category>
+        <span>
+          Whatever is good for your body and mind which helps you to feel happy
+          and relaxed.
+        </span>
+      </Description>
+
+      <Description>
+        <Category>Obligatory time </Category>
+        <span>
+          Everything that is mandatory. This is your morning routing, your
```

Comment:
also just a tiny typo: I think you mean 'morning routine' ? ;)

Reply:
ahaha too much into routers ! Thanks